tests/functional/spawn-on-demand/09-set-outputs.t

```diff
@@ -16,7 +16,7 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #-------------------------------------------------------------------------------
 
-# Check that "cylc set-outputs" works like it says on the tin.
```

Comment:
Given that these are reftests, they would be easy wins for Oliver's new reftest-integration testing framework?

Reply:
Possibly yes, but could be a follow-up.